.travis.yml

```diff
@@ -1,73 +0,0 @@
-dist: xenial
-sudo: required
-
-services:
-  - docker
-
-cache:
-  directories:
-    - $HOME/.m2
-    - $HOME/.npm
```

Comment:
I'm guessing the `$HOME/.npm` caching was a mistake / is not actually required/used?

Reply:
Yes. We don't use npm.